frontend/package.json

```diff
@@ -20,6 +20,7 @@
     "react-hot-loader": "4.12.20",
     "react-is": "16.13.1",
     "react-redux": "7.2.0",
+    "react-router-dom": "^6.2.2",
```

Comment:
Could not figure out how to successfully use router.

Reply:
Since I could not figure out how to use routes for the Recipe page, I only used the Home page to display all information.